### ISSUE TITLE
Does SuperSLiM supports sub headers ?

### ISSUE THREAD

**Issue by d3bt3ch** [TonicArtos/SuperSLiM#225]
Does SuperSLiM supports sub headers / sub sections ?

**Comment by TonicArtos** [comment]
Yes. The master branch has added support for this, but it isn't feature complete.

**Comment by d3bt3ch** [comment]
When would SuperSLiM 5 be released ? Anxiously waiting for the release.

**Comment by TonicArtos** [comment]
Fair enough, I've left you guys hanging for a long time. I've personally been using the current release for about a year now. I haven't need any more than what has been done for now, which is why sticky footers was done but not sticky headers.

I suppose I can get all the header variations done today and tomorrow. Grid view will take longer. Actually I need it in a upcoming project so I guess I have an imperative too.

**Comment by TonicArtos** [comment]
Well, it has been two weekends working here and there on the sticky headers. I've gotten it mostly working, but I am still ironing out bugs. I thought it would be a lot more straight forward than this.

**Comment by d3bt3ch** [comment]
No worries :) When can we expect a release ?

**Comment by TonicArtos** [comment]
Based on where I am and how long it has taken me to get here, I think I'll be able to get out the sticky headers component sometime between this weekend and mid next week. It depends on how much time I get free, and how long it takes me to eliminate the last of these incremental and intermittent frame by frame corner cases and bugs. I thought it was going to be just a knock out of the park, but it turns out scrolling nested sticky headers is a real pain.

**Comment by cherrydev** [comment]
Thanks a lot for all your hard work. I only just discovered SuperSLiM and sticky headers is actually exactly what I'm looking for.

**Comment by d3bt3ch** [comment]
@TonicArtos Thanks a lot for all your hard work. Waiting for 5 release.

**Comment by d3bt3ch** [comment]
@TonicArtos Any update on when can we expect version 5 ?

**Comment by d3bt3ch** [comment]
Any update on version 5 release date ?
